### DISCUSSION TITLE
I don't know how to thank you enough

### DISCUSSION THREAD

**Discussion by sethnielson** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#78] [General]
Thank you so much. I had to upgrade to Windows 11 but this inability to drag files to the taskbar was killing me. 

do you have a donations or support mechanism?

**Comment by HerMajestyDrMona** [comment]
I'm happy that you like the program :) Thank you for the nice feedback,. It's very rare to see positive messages from people who don't report any issues at the same time.

There is a donation possibility, described at the [bottom of the README page](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#donation). Generally it redirect to a [website](https://ee2.eu/overview/#donate) with my other project, where the paypal donation button is available. I should have probably added an unique button for Windows11DragAndDropToTaskbarFix, but I was not finding it as a priority and didn't want to give a false impression that the program was designed for profit.

Thanks again for your intention to support me.

**Reply by olcariz** [reply]
You are saving lifes up there sir, keep up the good work !

**Reply by olcariz** [reply]
And try to be hired by Microsoft, or at least teach them how to maintain their os :)

**Comment by sethnielson** [comment]
Donation made. Thank you.

On Fri, May 20, 2022 at 1:58 PM Dr.MonaLisa ***@***.***>
wrote:

> I'm happy that you like the program :) Thank you for the nice feedback,.
> It's very rare to see positive messages from people who don't report any
> issues at the same time.
>
> There is a donation possibility, described at the bottom of the README
> page
> <https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#donation>.
> Generally it redirect to a website <https://ee2.eu/overview/#donate> with
> my other project, where the paypal donation button is available. I should
> have probably added an unique button for Windows11DragAndDropToTaskbarFix,
> but I was not finding it as a priority and didn't want to give a false
> impression that the program was designed for profit.
>
> Thanks again for your intention to support me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/discussions/78#discussioncomment-2793278>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEYSE3VTV5EL6CA73UEUTRLVK7OETANCNFSM5WQA2USQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/repo-discussions/78/comments/2793278
> @github.com>
>


-- 
Seth James Nielson, PhD
   Founder & Chief Scientist
   Crimson Vista, Inc.
   ‪(512) 387-4310
--
IMPORTANT - This electronic message transmission and any attachment
contains information that may be confidential or privileged.  The
information contained herein is intended solely for the recipient and use
by any other party is not authorized.  If you are not the intended
recipient (or otherwise authorized to receive this message by the intended
recipient), any review, disclosure, copying, distribution, or other use of,
or taking of action in reliance upon, the contents of the information is
prohibited.  If you have received this electronic message transmission in
error, please contact the sender by reply email and delete all copies of
this message.  Crimson Vista Inc. accepts no responsibility for any loss or
damage resulting directly or indirectly from the use of this email or its
contents.  Thank You.
